meta/plants.tmpl

```diff
@@ -8,33 +8,45 @@
 # species.production_name	_PROD_NAME_
 species.common_name	_COMMON_NAME_
 
-# _GENBANK_ACCESSION_
-# https://www.ncbi.nlm.nih.gov/assembly/_GENBANK_ACCESSION_
-# https://_REFSEQ_FTP_URL_
+# _GENOME_ACCESSION_
+# https://www.ncbi.nlm.nih.gov/assembly/_GENOME_ACCESSION_
+# _GENOME_FTP_URL_
+# Assembly provider: _ASSEMBLY_PROVIDER_NAME_
+# Annotation provider: _ANNOTATION_PROVIDER_NAME_
 # RefSeq Annotation Name: _REFSEQ_ANN_NAME_
-# RefSeq Annotation Url: _REFSEQ_ANN_REPORT_URL_
+# RefSeq Annotation URL: _REFSEQ_ANN_REPORT_URL_
 
-#CONF	ASM_URL	_REFSEQ_FTP_URL_
+#CONF	ASM_URL	_GENOME_FTP_URL_
+
+# Community GFF3 loading
+#no CONF	GFF_FILE	_ANNOTATION_GFF3_
```

Comment:
should it be prefixed with no? disabled by default?

Reply:
Now that you mention it, for plants it is more frequent than not to load community annotations, so will remove the `no`. Cheers!